### DISCUSSION TITLE
Edit map together at the same time

### DISCUSSION THREAD

**Discussion by ZeriBun** [ppy/osu#14556] [Ideas]
i think it would be awesome for me and many others to be able to edit a map at the same time together somehow.
 so that people dont need to stream to each other to show what theyre doing when exchanging during collab mapping.
 this would also help alot when experienced mappers try to teach a newcomer to mapping

**Comment by sr229** [comment]
I guess something of a system like Visual Studio Live Share would work but I don't know we can use the same codebase for Multiplayer for live collab mapping.

**Comment by abraker95** [comment]
Associated feature request on forums https://osu.ppy.sh/community/forums/topics/108829
